From 7bf60cb66e91f5f74a6ee635fc5da97bdb59aad0 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Thu, 03 Feb 2022 08:05:40 +0200
Subject: [PATCH] [SW-68803] arcboot: make bringup pcie weak

Due to major changes in gaudi3 (usage of image pointers to extract
image properties) compared to gaudi2, the data processing and
validation is completely different. Hence making the function
to bringup pcie images as 'weak' in preparation to make this
function project specific for gaudi3 whilst maintaining backward
compatibility for existing projects.

Change-Id: I9bf880a0bf3d3a72aad23d1ccc6967511ea6f2ba
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index 838068d..878e4cf 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -361,7 +361,7 @@
 
 __weak void arcboot_rom_handle_tpm(void) {}
 
-static void arcboot_rom_bringup_pcie(void)
+__weak void arcboot_rom_bringup_pcie(void)
 {
 	void (*ppboot)();
 
